src/components/chip-set/chip-set.scss

```diff
@@ -69,7 +69,7 @@ $height-of-chip-set-input: pxToRem(36);
 limel-icon.mdc-chip__icon.mdc-chip__icon--leading {
     background-color: var(--icon-background-color, transparent);
     margin: 0 !important;
-    color: var(--icon-color, rgba(0, 0, 0, 0.54));
+    color: var(--icon-color, rgb(var(--contrast-1100)));
```

Comment:
This is the only change to the component itself, the rest is just changes to the examples. So I don't think "prevent default chips to appear visually so dominant" is a good description of the change here… I'll come up with something better, but I'm making this comment because I'll be logging of for the weekend now 🙂

Reply:
OK :)

Reply:
Haha, I had _totally_ forgotten about this, so good thing I wrote that comment, and thanks for commenting back on it, so that I saw it again 😂 🙏

Reply:
⚡